.github/workflows/codecov_ats.yml

```diff
@@ -0,0 +1,105 @@
+
+name: IGNORE ME codecov-ats
+
+on:
+  pull_request:
+
+env:
+    CLI_VERSION: v0.1.5
+
+    # Cancel in progress workflows on pull_requests.
+    # https://docs.github.com/en/actions/using-jobs/using-concurrency#example-using-a-fallback-value
+    concurrency:
+    group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+    cancel-in-progress: true
+
+jobs:
+    files-changed:
+        name: detect what files changed
+        runs-on: ubuntu-20.04
+        timeout-minutes: 3
+        # Map a step output to a job output
+        outputs:
+          api_docs: ${{ steps.changes.outputs.api_docs }}
+          backend: ${{ steps.changes.outputs.backend_all }}
+          backend_dependencies: ${{ steps.changes.outputs.backend_dependencies }}
+          backend_any_type: ${{ steps.changes.outputs.backend_any_type }}
+          migration_lockfile: ${{ steps.changes.outputs.migration_lockfile }}
+          plugins: ${{ steps.changes.outputs.plugins }}
+        steps:
+          - uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8  # v3.1.0
+
+          - name: Check for backend file changes
+            uses: getsentry/paths-filter@4512585405083f25c027a35db413c2b3b9006d50  # v2.11.1
+            id: changes
+            with:
+              token: ${{ github.token }}
+              filters: .github/file-filters.yml
+    coverage-ats:
+        # Temporary test
+        if: needs.files-changed.outputs.backend == 'true'
+        needs: files-changed
+        timeout-minutes: 40
+        runs-on: ubuntu-latest
+        strategy:
+          # This helps not having to run multiple jobs because one fails, thus, reducing resource usage
+          # and reducing the risk that one of many runs would turn red again (read: intermittent tests)
+          fail-fast: false
+          matrix:
```

Comment:
The python tests are broken up into multiple shards to try and reduce the run time, can that be skipped here?

Mainly asking since adding this action as is will take up another 5 concurrent actions which I'd like to avoid if we can.

Reply:
Replacing `pg-version: ${{ matrix.pg-version }}` with `pg-version: 14` and removing the `matrix:` and `env:` parameters should do it for you.